python/fledge/services/core/server.py

```diff
@@ -67,6 +68,10 @@
 
 SSL_PROTOCOLS = (asyncio.sslproto.SSLProtocol,)
 
+SERVICE_JWT_SECRET = 'f0gl@mp+Fl3dG3'
```

Comment:
The secret is not very secret if it is in a public repository. I wonder if we could generate a random secret at build time in much the same way as we generate the version.h.

Reply:
We have a similar one in python/fledge/services/core/user_model.py and that is for user authentication.

A possible new solution will address both tokens generation

Reply:
please add a TODO

Reply:
Please add the TODO here